### DISCUSSION TITLE
Presa Shelly non più visibile

### DISCUSSION THREAD

**Discussion by bebbbo** [indomus/forum#99] [HUB: Home Assistant]
Ciao a tutti, mi sono sparite le prese Shelly da home assistant, forse ho capito il motivo, ma mi pare strano che non ci sia una soluzione, vi spiego.

A casa mia ho una Fibra Ottica Vodafone, con la Vodafone Station a cui ho disinstallato il WIFI e su una delle 4 porte Ethernet ho collegato il sistema Wifi Mesh di Netgear ORBI che mi distribuisce in casa la connessione internet via WIFI.
Sulla Vodafone Station ho poi collegato, via cavo, l'hub della somfy per far funzionare le tapparelle elettriche.
La vodafone Station assegna ai device collegati un indirizzo IP del tipo 192.168.0.X, mentre il router Netgear ORBI assegna un IP del tipo 10.0.0.X, quindi tutti gli apparecchi connessi via WIFI hanno un IP nella rete locale del tipo 10.0.0.X.
Inizialmente il Raspberry PI con Home Assistant era connesso via cavo direttamente al router ORBi e quindi aveva un indirizzo IP 10.0.0.X.
Con questa configurazione mi funzionava tutto, vedevo su Home Assistant la presa Shelly che aveva indirizzo IP 10.0.0.3 e vedevo anche le tapparelle che erano connesse all'HUB Somfy che invece aveva indirizzo IP 192.168.0.X perchè connesso via cavo alla vodafone station.
Ora ho fatto una semplice modifica, ho connesso il Raspberry PI con Home Assistant via cavo a monte, ovvero direttamente alla Vodafone Station e gli è stato quindi assegnato un IP 192.168.0.X e non vedo più la presa Shelly che ha sempre IP 10.0.0.3.
Ho riavviato, ho anche reinstallato l'integrazione Shelly Smart Home, ma nulla.
Dipende secondo voi da questo spostamento che ho fatto?
Come risolvo?

P.S. contestualmente allo spostamento avevo anche installato DUCKDNS e configurato l'accesso via internet a home assistant, che funziona correttamente.

Grazie

**Comment by poliant** [comment]
ciao @bebbbo  direi che il problema sta nel fatto che la shelly ed il raspberry sono su due rete differenti e non hai regole nelle tabelle di routing per poter instradare tutti i pacchetti tra le due reti.
se ti colleghi al raspi riesci a raggiungere la presa shelly? tipo se dal raspi fai un ping alla shelly (o un traceroute) che risultato hai?
Oppure se sei collegato alla vodafonestatio con il cell o il PC, riesci a raggiungere la shelly?

Se non riesci in entrambi i casi, vuol dire che devi aggiornare la tabella di routing della vodafone station.
Se invece nel primo caso non riesci e nel secondo si, devi controllare che il raspi abbia configurato correttamente il gateway e dns

**Reply by bebbbo** [reply]
Ciao e grazie per il feedback.

Ho risolto nel modo più semplice, nel senso che ho ricollegato il raspberry direttamente via cavo al Netgear ORBI che mi crea la rete wifi in casa e rivede ora tutte le periferiche e i sensori collegati a questa sotto rete.
Ora il problema però è un altro, non mi funziona più l'accesso in remoto tramite duckdns perchè il raspberry è nella sottorete del netgear.
Per ricapitolare la mia connessione/rete di casa è fatta cosi:

VODAFONE STATION (collegata alla fibra ottica) che ha IP 192.168.1.1 e assegna ai device ad essa collegati degli IP del tipo 192.168.1.X

NETGEAR ORBI WIFI MESH (collegato via cavo alla vodafone station) a cui è assegnato l'IP 192.168.1.6 da parte della Vodafone Station

RASPBERRY PI CON HASSIO (collegato via cavo al netgear orbi) a cui è assegnato l'IP 10.0.0.113 da parte del Netgear ORBI

In questo modo Home Assistant mi vede tutto, però non accedo più via internet sull'indirizzo duckdns.
Sulla Vodafone Station nel Port Forwarding ho aperto la porta locale 8123 con porta pubblica 443 (per non stare poi sempre a specificare la porta quando digito l'url) sull'IP 192.168.1.6 che è quello assegnato al netgear Orbi, anche perchè non me lo fa aprire sull'IP 10.0.0.113 perchè non lo riconosce.

Ma cosi non mi funziona, come posso risolvere?

Grazie

**Reply by poliant** [reply]
se ti colleghi alla vodafone station, riesci ad accedere ad HA? se si, che indirizzo usi (10.0.0.113:8123 oppure 192.168.1.6:8123)?
che regole hai imposto sul netgear di inoltro tra le due reti (192.168.1.x e 10.0.0.x)? Prova a mettere il port forward sul netgear da 192.168.1.6 a 10.0.0.113

**Comment by Scialla** [comment]
Io ho la situazione opposta (e più incasinata):
FTTC-modem tiscali-time capsule che da ip 192.168.1.x a cui è collegato tutto, tranne uno shelly e un pzem004t, collegati a un repetear che da ip su range 10.0.0.x.
Ovviamente non raggiungo i due device su rete da quella principale ma, essendo configurati in mqtt, i dati filtrano tra la sottorete e la rete principale e li vedo e comando via mqtt, senza avere aggiornato alcuna tabella di routing, manco io so perché :D

**Comment by bebbbo** [comment]
Io alla fine ho risolto nel modo più semplice, ho cambiato il fuzionamento del Netgear da Router ad Access Point e quindi la vodafone station assegna gli IP a tutti (192.168.1.X), anche a quelli che si collegano al wifi del Netgear.

In fondo non mi serviva a nulla aver il netgear che creava una sottorete (con tutti i casini che ne derivavano).
